yazi-adapter/src/image.rs

```diff
@@ -73,13 +73,13 @@ impl Image {
 			.unwrap_or((PREVIEW.max_width, PREVIEW.max_height))
 	}
 
-	pub(super) fn pixel_area(size: (u32, u32), rect: Rect) -> Rect {
+	pub(super) fn pixel_area((width, height): (u32, u32), rect: Rect) -> Rect {
```

Comment:
Please keep the original code here as `size` is a field term in the `yazi-adapter` scope